server/CGameHandler.h

```diff
@@ -248,14 +225,11 @@ class CGameHandler : public IGameCallback, public CBattleInfoCallback, public En
 	template <typename Handler> void serialize(Handler &h, const int version)
 	{
 		h & QID;
-		h & states;
-		h & battles;
-		h & heroPool;
 		h & getRandomGenerator();
-		h & playerMessages;
-
-		if (!h.saving)
-			deserializationFix();

```

Comment:
Did you check that game over network works well? I remember there were some issues with stacks of heros because of this deserialization fix but don't remember details

Reply:
No, this is different deserializationFix, not related to stacks/bonus system. Check CGameHandler.cpp